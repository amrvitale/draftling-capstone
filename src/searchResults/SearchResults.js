import React from 'react';
import { Link } from 'react-router-dom';
import Search from '../search/Search';
class SearchResults extends React.Component {
    render() {
        return (
            <div className="searchResults">
                <h1>Search Results</h1>
                <section className="resultsPlaceholder">
                <Link to='/read'>
                    <button type="submit">Read</button>
                </Link>

                </section>
                
                <Link to='/search'>
                    <button type="submit">Return to Search</button>
                </Link>
            </div>
        );
    }
}

export default SearchResults;

//except read link will take you to id of story yes?