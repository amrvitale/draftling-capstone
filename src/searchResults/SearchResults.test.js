import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import SearchResults from './SearchResults'

describe (`SearchResults component`, () => {
    it('renders a .SearchResults by default', () => {
        const wrapper = shallow(<SearchResults />)
        expect (toJson(wrapper)).toMatchSnapshot()
    })

    
})