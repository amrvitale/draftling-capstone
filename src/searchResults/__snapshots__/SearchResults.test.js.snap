// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResults component renders a .SearchResults by default 1`] = `
<div
  className="searchResults"
>
  <h1>
    Search Results
  </h1>
  <br />
  <br />
  <section
    className="resultsPlaceholder"
  >
    <Link
      to="/read"
    >
      <button
        type="submit"
      >
        Read
      </button>
    </Link>
  </section>
  <Link
    to="/search"
  >
    <button
      type="submit"
    >
      Return to Search
    </button>
  </Link>
</div>
`;
